uberfire-extensions/uberfire-wires/uberfire-wires-core/uberfire-wires-core-grids/src/main/java/org/uberfire/ext/wires/core/grids/client/model/GridData.java

```diff
@@ -295,6 +295,39 @@ void collapseCell(final int rowIndex,
     void expandCell(final int rowIndex,
                     final int columnIndex);
 
+    /**
+     * Update width for column with {@link GridColumn.ColumnWidthMode#auto}
+     * @return a boolean that indicates if grid need to be redraw or not
+     */
+    boolean refreshWidth();
+
+    /**
+     * Update width for column with {@link GridColumn.ColumnWidthMode#auto}
+     * @param currentWidth is the grid width before this resize iteration
+     * @return a boolean that indicates if grid need to be redraw or not
+     */
+    boolean refreshWidth(double currentWidth);
+
+    /**
+     * Update visible size information and refresh column's width
```

Comment:
"Update visible size information and refresh columns width. See {@link GridData#refreshWidth()}".

Just wondering if either linking to the "refresh" methods or adding the same explicit text here would make it clearer?

Reply:
Done